src/ndb.rs

```diff
@@ -429,6 +430,63 @@ impl Ndb {
         ))
     }
 
+    pub fn search_profile(
+        &self,
+        transaction: &Transaction,
+        search: &str,
+        limit: u32,
+    ) -> Result<Vec<[u8; 32]>> {
```

Comment:
nostrdb-rs apis should be non-copy


```suggestion
    pub fn search_profile<'a>(
        &self,
        transaction: &'a Transaction,
        search: &str,
        limit: u32,
    ) -> Result<Vec<&'a [u8; 32]>> {
```

Reply:
> nostrdb-rs apis should be non-copy. ndb_search only holds a pointer to the pubkey in the key in nostrdb.

right, of course